.github/workflows/zephyr.yml

```diff
@@ -18,14 +18,11 @@ jobs:
         # many lines. Pay attention to COMMAS.
         IPC_platforms: [
           # - IPC3 default
-          apl cnl,
-          icl jsl,
-          tgl tgl-h,
           imx8 imx8x imx8m,
           # - IPC4 default
           mtl,
           # Very few IPC3 platforms support IPC4 too.
-          -i IPC4 tgl,
+          -i IPC4 tgl
```

Comment:
YAML allows trailing commas to avoid git noise (like this), don't remove it.
See example 2.6 in the spec https://yaml.org/spec/1.2.2/#21-collections

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas